src/GHC.hs

```diff
@@ -131,8 +131,10 @@ programPath context@Context {..} = do
     -- The @touchy@ utility lives in the @lib/bin@ directory instead of @bin@,
```

Comment:
Could you update the comment as well?

Reply:
I added "Likewise for unlit" 2-3 lines below, as a comment to the same function, is that unsatisfactory / not clear enough?

Reply:
Ah, indeed, I didn't notice.